src/webgpu/api/validation/encoding/queries/resolveQuerySet.spec.ts

```diff
@@ -1,13 +1,94 @@
 export const description = `
-TODO:
-- invalid GPUQuerySet
-- firstQuery and/or queryCount out of range
-- invalid destination buffer
-- destinationOffset out of range
-- ?
+Validation tests for resolveQuerySet.
 `;
```

Comment:
nit: It would have been helpful to have a first PR which wrote an expanded plan but didn't start implementing tests. It would have caught these issues more quickly. Though I know the review latency makes this harder, so it wasn't really a problem for these relatively simple tests.